File: .templates/template.py
# run with 'py template.py'

import sys
import math

def solve_part_one(values):
    return 0

def solve_part_two(values):
    return 0

def main():
    with open('input.txt', 'r') as f:
        if f.mode == 'r':
            values = [line.strip() for line in f.readlines()]
            print(solve_part_one(values))
            print(solve_part_two(values))

if __name__ == '__main__':
    main()

File: .templates/template.cpp
// run with "g++ template.cpp -o template && ./template"

#include <fstream>
#include <iostream>
#include <vector>

using namespace std;

vector<string> values;

int solve_part_one() {
    return 0;
}

int solve_part_two() {
    return 0;
}

int main() {

  ifstream f;
  f.open("input.txt");
  string line;

  while (!f.eof()) {
    getline(f, line);
    values.push_back(line);
  }

  cout << solve_part_one() << endl;
  cout << solve_part_two() << endl;

  return 0;

}
